packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -124,3 +124,5 @@ export default function EditorStyles( { styles, scope } ) {
 		</>
 	);
 }
+
+export default memo( EditorStyles );
```

Comment:
This is kind of unrelated but I noticed the component renders 5 times while the block preview is computing the right "height".